R/standardize_subject_ids.R

```diff
@@ -152,23 +137,23 @@ check_subject_ids <- function(data,
 #' # perform the correction
 #' dat <- correct_subject_ids(
 #'   data             = dat,
-#'   id_column_name   = "study_id",
+#'   target_columns   = "study_id",
 #'   correction_table = correction_table
 #' )
-correct_subject_ids <- function(data, id_column_name, correction_table) {
+correct_subject_ids <- function(data, target_columns, correction_table) {
   checkmate::assert_data_frame(correction_table, any.missing = FALSE,
                                min.rows = 1L, ncols = 2L, null.ok = FALSE)
 
   stopifnot("Some ids in the correction table were not found in the input data"
-            = all(correction_table[["from"]] %in% data[[id_column_name]]))
+            = all(correction_table[["from"]] %in% data[[target_columns]]))
 
   # perform the substitution
   idx                         <- match(correction_table[["from"]],
-                                       data[[id_column_name]])
-  data[[id_column_name]][idx] <- correction_table[["to"]]
+                                       data[[target_columns]])
+  data[[target_columns]][idx] <- correction_table[["to"]]
 
   # check whether substitution did not introduce any duplicate
-  data                        <- check_subject_ids_oness(data, id_column_name)
+  data                        <- check_subject_ids_oness(data, target_columns)
```

Comment:
Consider renaming this function `check_unique_ids`

Reply:
@bahadzie I want functions in this module to either end with `subject_ids` and contain it. 

That way, it will be easy to cite them in pkgdown.